frontend/server/handlers/pod-logs.ts

```diff
@@ -73,7 +73,13 @@ export function getPodLogsHandler(
 
     try {
       const stream = await getPodLogsStream(podName, podNamespace);
-      stream.on('error', err => res.status(500).send('Could not get main container logs: ' + err));
+      stream.on('error', err => {
+        if (err.message.indexOf('Unable to find pod log archive information') > -1) {
```

Comment:
nit: `err?.message && err.message.indexOf`?
because someday maybe an err without message could be thrown